### ISSUE TITLE
after idle sent, a blank command being sent too

### ISSUE THREAD

**Issue by si458** [emailjs/emailjs-imap-client#78]
for some reason when the client issues the idle command, it is also sending a blank command right after

SERVER:
- Tue, 09 Feb 2016 14:54:18 -> 10.0.0.222 -> Success: Action=[Recv Command], Details=[W41 IDLE]
- Tue, 09 Feb 2016 14:54:18 -> 10.0.0.222 -> Success: Action=[Sent Command], Details=[+ IDLE accepted, awaiting DONE]
- Tue, 09 Feb 2016 14:54:18 -> 10.0.0.222 -> Success: Action=[Start IDLE State]
- Tue, 09 Feb 2016 14:54:18 -> 10.0.0.222 -> Success: Action=[Recv Command]
- Tue, 09 Feb 2016 14:54:18 -> 10.0.0.222 -> Success: Action=[Sent Command], Details=[* BAD Protocol Error: Empty command.]
- Tue, 09 Feb 2016 14:55:18 -> 10.0.0.222 -> Success: Action=[Recv Command], Details=[DONE]
- Tue, 09 Feb 2016 14:55:18 -> 10.0.0.222 -> Success: Action=[Sent Command], Details=[ OK IDLE completed.]

CLIENT:
- [DEBUG][2016-02-09T16:19:40.971Z][undefined] Client started idling
- [DEBUG][2016-02-09T16:19:40.972Z][undefined] Entering idle with IDLE
- [DEBUG][2016-02-09T16:19:40.972Z][undefined] C: W44 IDLE
- [DEBUG][2016-02-09T16:19:41.043Z][undefined] S: * BAD Protocol Error: Empty command.

**Comment by felixhammerl** [comment]
ok, i don't really have a clue why that is right now, but i'll look into it when i have more time in the next couple of days.

**Comment by AndryBray** [comment]
Hi @felixhammerl 
any news about this issue?
I'd appreciate even some workaround to apply in the meanwhile. Thank you

**Comment by felixhammerl** [comment]
oops, somehow this slipped my radar. i somehow thought this would magically be handled by https://github.com/emailjs/emailjs-imap-client/issues/77 ... i'll put it on my radar for this weekend...

**Comment by AndryBray** [comment]
thank you! I hope it will be simple to fix

**Comment by felixhammerl** [comment]
ok, so i've finally found time to tackle this. good news, reproducing it is trivial and it also happens on node:

```
var ImapClient = require('emailjs-imap-client');

var client = new ImapClient('imap.gmail.com', 993, {
    auth: {
        user: 'USER',
        pass: 'PASS'
    }
});

client.connect().then(() => {
    console.log('### CONNECTED');
    return client.listMailboxes();
}).then(() => {
    console.log('### MAILBOXES');
    setTimeout(() => {
        client.listMailboxes();
    }, 20000)
});
```

**Comment by felixhammerl** [comment]
oops.

https://github.com/emailjs/emailjs-imap-client/blob/master/src/emailjs-imap-client-imap.js#L425

**Comment by felixhammerl** [comment]
fixed version published as v2.0.2

sorry that took so long :(
